app/index.html

```diff
@@ -0,0 +1,675 @@
+                            d="M545.695,787.175l-79.486-54.983c-0.179-0.124-0.349-0.256-0.512-0.393l-0.646,1.004  c-1.745,2.714-0.991,6.415,1.731,8.298l71.119,49.195c4.333,2.997,9.76,2.625,12.025-0.897l0.17-0.265  C548.603,788.79,547.098,788.146,545.695,787.175z" />
+                    <path
+                            id="XMLID_86_"
+                            style="fill:#FF6161;"
+                            d="M341.033,504.79h-16.177c-8.246,0-14.93-6.684-14.93-14.93v-44.076h46.037v44.076  C355.964,498.106,349.279,504.79,341.033,504.79z" />
+                    <path
+                            id="XMLID_112_"
+                            style="fill:#F4A36C;"
+                            d="M315.066,451.801c-10.677,0-19.333,8.656-19.333,19.333s8.656,19.333,19.333,19.333  s19.333-8.656,19.333-19.333S325.744,451.801,315.066,451.801z M315.067,486.218c-8.317,0-15.083-6.767-15.083-15.083  c0-8.317,6.766-15.083,15.083-15.083c8.317,0,15.083,6.766,15.083,15.083C330.15,479.451,323.384,486.218,315.067,486.218z" />
+                    <path
+                            id="XMLID_97_"
+                            style="fill:#F7BC8B;"
+                            d="M341.033,504.79h-16.177c-8.246,0-14.93-6.684-14.93-14.93v-44.076h46.037v44.076  C355.964,498.106,349.279,504.79,341.033,504.79z" />
+                    <path
+                            id="XMLID_108_"
+                            style="fill:#EFA573;"
+                            d="M331.706,490.79c-4.564,0-8.264-3.7-8.264-8.264v-36.742h-13.516v44.076  c0,8.246,6.685,14.93,14.93,14.93h16.177c7.932,0,14.401-6.19,14.883-14H331.706z" />
+                    <path
+                            id="XMLID_18_"
+                            style="fill:#FC9F6F;"
+                            d="M522.446,504.79c0.961-2.596,1.488-5.403,1.488-8.333c0-13.255-10.745-24-24-24  c-13.255,0-24,10.745-24,24c0,2.93,0.527,5.737,1.488,8.333H522.446z" />
+                    <path
+                            id="XMLID_44_"
+                            style="fill:#F05364;"
+                            d="M393.346,391.051" />
+                    <circle
+                            id="XMLID_12_"
+                            style="opacity:0.77;fill:#F2F2F2;"
+                            cx="327.927"
+                            cy="431.278"
+                            r="6.472" />
+                    <circle
+                            id="XMLID_43_"
+                            style="opacity:0.77;fill:#F2F2F2;"
+                            cx="339.863"
+                            cy="408.62"
+                            r="9.713" />
+                    <circle
+                            id="XMLID_45_"
+                            style="fill:#65706C;"
+                            cx="244.986"
+                            cy="328.079"
+                            r="27.606" />
+</svg>
+                <i class="fas fa-angle-down fa-5x moreContent"></i>
```

Comment:
this doesn't "jump" down the page when clicked - idk if it's supposed to?

Reply:
it wasn't supposed to but i think it's a good idea i shall add it in! :)